examples/spring-boot-webflux/src/main/java/example/springframework/boot/webflux/HelloAnnotatedService.java

```diff
@@ -0,0 +1,26 @@
+package example.springframework.boot.webflux;
+
+import javax.validation.constraints.Size;
+
+import org.springframework.stereotype.Component;
+import org.springframework.validation.annotation.Validated;
+
+import com.linecorp.armeria.server.annotation.ExceptionHandler;
+import com.linecorp.armeria.server.annotation.Get;
+import com.linecorp.armeria.server.annotation.Param;
+
+@Component
+@Validated
+@ExceptionHandler(ValidationExceptionHandler.class)
+public class HelloAnnotatedService {
```

Comment:
Could you add a short Javadoc which explains this is not a Webflux-based service but an annotated service that leverages Armeria's built-in annotations? We could also add a link to our official documentation about annotated services.